api/src/test/java/org/pmiops/workbench/firecloud/FireCloudServiceImplTest.java

```diff
@@ -197,21 +190,6 @@ public void testNihCallbackServerError() throws Exception {
     service.postNihCallback(any());
   }
 
-  @Test
```

Comment:
For me, the value in this kind of test is just live documentation rather than coverage. Also, if there are any failure modes (like when failing to refresh), it might be good to demonstrate what is expected to happen.